apps/landingPage/src/components/Headline.js

```diff
@@ -13,87 +13,137 @@ import airport from '../images/airport.jpg';
-            <img src={kiwiLogo} height={43} alt="kiwi_logo" />
-            <img src={reactLogo} height={45} alt="react_logo" />
-            <img src={tequilaLogo} height={43} alt="tequila_logo" />
-            <img src={graphqlLogo} height={43} alt="graphql_logo" />
-          </ContainerIcons>
-        </ContainerTop>
-        <ContainerBottom>
-          <ContainerLeftText>
-            <Heading>Get started</Heading>
-            <WrapperText>
-              <Text size="large">
-                PLACEHOLDER!!!!!You've had a look at Tequila, but you've
-                preffered yourTequila Tequila mixed with something else. Just
-                grab Margarita.You've had a look at Tequila, but you've always
-                preffered your Tequila mixed with something else.
+          </Text>
+        </HeadlineTextContainer>
+        <Container>
+          <ContainerTop>
+            <HeaderAndDesignWrapper>
+              <BackgroundGradientAccent>
+                <GreenRectangle />
+              </BackgroundGradientAccent>
+              <Text align={alignment}>
+                <Heading type="title2">
+                  You've had a look at Tequila, but you've always preffered your
+                  Tequila mixed with something else. Just grab a Margarita.
+                </Heading>
               </Text>
-            </WrapperText>
-            <WrapperButtons>
-              <Button>
-                <LinkNoStyle
-                  href={margarita}
-                  color={defaultTokens.paletteWhite}
-                >
-                  Get started
-                </LinkNoStyle>
-              </Button>
-              <WrapperRightButton>
-                <Button type="white">
+            </HeaderAndDesignWrapper>
+            <ContainerIcons>
+              <img src={kiwiLogo} height={43} alt="kiwi_logo" />
+              <img src={reactLogo} height={45} alt="react_logo" />
+              <img src={tequilaLogo} height={43} alt="tequila_logo" />
+              <img src={graphqlLogo} height={43} alt="graphql_logo" />
+            </ContainerIcons>
+          </ContainerTop>
+          {isTablet && (
+            <>
+              <MobileGradientAccent>
+                <PlaneImageMobile />
+              </MobileGradientAccent>
+            </>
+          )}
+          <ContainerBottom>
+            <ContainerLeftText>
+              <Heading>Get started</Heading>
+              <WrapperText>
+                <Text size="large">
+                  PLACEHOLDER!!!!!You've had a look at Tequila, but you've
+                  preffered yourTequila Tequila mixed with something else. Just
+                  grab Margarita.You've had a look at Tequila, but you've always
+                  preffered your Tequila mixed with something else.
+                </Text>
+              </WrapperText>
+              <WrapperButtons>
+                <Button block={isTablet || false} size={buttonSize}>
                   <LinkNoStyle
```

Comment:
I would suggest rename it to more readable form `LinkWithoutStyle`

Reply:
Alright, will rename it